### ISSUE TITLE
Unable to use ipad as a secondary display

### ISSUE THREAD

**Issue by jufel** [ClassicOldSong/Widecar#5]
I tried all the troubleshooting steps, but my windows is just mirroring the display, and 'Extend' doesn't seem to work. What should I do?

**Comment by ClassicOldSong** [comment]
Can you confirm that the virtual display is created correctly in the system settings?

You can also try https://github.com/ClassicOldSong/Apollo/releases/tag/apollo-0.0.0-preview to see if it works for you.

**Comment by ClassicOldSong** [comment]
You need to install Moonlight on your iPad in order to use Sunshine/Apollo.

**Comment by jufel** [comment]
I use Sunshine and Moonlight on the iPad.

**Comment by ClassicOldSong** [comment]
Then you need to start streaming on your iPad using Moonlight.

**Comment by jufel** [comment]
Yes i did that and its only mirroring the windows screen; even when I click extend. Ive also tried the Troubleshooting method for the same problem, but no changes still.

**Comment by ClassicOldSong** [comment]
So have you seen any virtual screen created temporarily when the stream starts?

Or can you try Apollo as I mentioned above to see if it works?

**Comment by jufel** [comment]
I do have Parsec VDD installed...but i dont know how to see if any virtual screen is created or not.

**Comment by ClassicOldSong** [comment]
Could you just try Apollo and see if it works?

By the way what version of  Windows are you using?

**Comment by cdguy** [comment]
@ClassicOldSong I have the same issue.
Apollo only mirrors the PC screen to my ipad
I am using windows 11

And when I open display settings, I only see the screen of my laptop, so there is no virtual screen displayed

**Comment by ClassicOldSong** [comment]
If you have seen 1|2 in your Windows settings, then the only thing you need to do is make the second display Extended 

If not, you may have not got the vdd driver installed correctly.

If virtual displays can create but you're not able to launch any other apps, it's probably a permission issue: https://docs.lizardbyte.dev/projects/sunshine/en/latest/troubleshooting/windows.html

And this project is abandoned as described in the README, if you have issues with Apollo please open an issue there instead of here.
